fairscale/nn/__init__.py

```diff
@@ -5,6 +5,7 @@
 
 from .data_parallel import FullyShardedDataParallel, ShardedDataParallel
 from .misc import FlattenParamsWrapper
+from .misc.auto_wrap import auto_wrap, enable_wrap, wrap
```

Comment:
If I put `.auto_wrap import auto_wrap, enable_wrap, wrap` into `fairscale/nn/misc/__init__.py` it was complaining about a cycling import (FSDP). I'll try finnick around but let me know what's prefered